src/web/src/components/Posts/Post.tsx

```diff
@@ -251,6 +260,11 @@ const extractGitHubUrlsFromPost = (htmlString: string): string[] => {
   );
 };
 
+const extractVideoId = (post: Post): string => {
```

Comment:
When a function returns the result of an expression, and you use an arrow function, you don't need a function body:

```ts
const extractVideoId = (post: Post): string => post.guid.split(':')[2];
```